eclass/coreos-go.eclass

```diff
@@ -75,7 +79,7 @@ coreos-go_src_compile() {
 coreos-go_src_install() {
 	debug-print-function ${FUNCNAME} "$@"
 
-	dobin "${GOBIN}"/*
+	dobin ${GOBIN}/*
```

Comment:
Please leave quoted. Looks funny and in this case not strictly required but I prefer to err on the side of obsessively quoting when writing shell.